README.md

```diff
@@ -94,9 +122,9 @@ good installation path if you are just testing or doing development on the plugi
 
 * Enable Nexus console: edit `<nexus_dir>/bin/nexus.vmoptions` and change `karaf.startLocalConsole`  to `true`.
```

Comment:
Boyscout: Here's another instance of "Nexus" (should be NXRM).

Reply:
fixed 9256cd94a7e43a6f43ad6f942b7665f0468ccb7d